dashboard/app/mailers/pd/application/teacher1920_application_mailer.rb

```diff
@@ -68,7 +68,7 @@ def principal_approval_completed_partner(teacher_application)
       raise "Partner contact email is required, application id #{@application.id}" unless partner_contact_email
 
       mail(
-        from: 'Anthonette Peña <teacher@code.org>',
+        from: 'Liz Gauthier <teacher@code.org>',
```

Comment:
Adjusts the "from" header on three emails:

- `principal_approval_completed_partner__with_partner`
- `principal_approval_completed_partner__with_partner_no_contact`
- `principal_approval_completed_partner__without_partner`